redux/actionTypes.js

```diff
@@ -0,0 +1,9 @@
+// Action Types
+
+// Message
+export const ADD_MESSAGE = 'ADD_MESSAGE';
+
+// User
+export const SET_CURRENT_USER = 'SET_CURRENT_USER';
+export const SET_CURRENT_CHAT = 'SET_CURRENT_CHAT';
+export const ADD_CHATROOM = 'ADD_CHATROOM';
```

Comment:
i think we should agree on naming conventions... should we call it CHAT if it refers to a room and MESSAGE if it refers to a message within the chat? so here could be "ADD_CHAT" btw, i usually use "CREATE" instead of "ADD" to follow along with CRUD, so "CREATE_CHAT"

Reply:
Yeah, earlier in slack, I just thought it would be a good idea to have something so we can make these sort of adjustments. Agreed with your naming conventions, @mnicole. That basically follows the naming convention of the db as well, and that's a good point about CRUD.

Reply:
Agreed, CHAT is a good convention. I think the CRUD model is a good point but the the odd thing here is we really aren't creating these things--they're created in the database--we're just adding information about them to the Redux state, that's why I was thinking "Add". That hangup may just be me getting used to Redux--do you see people usually use that terminology with it? I could see it makes sense in that we're creating information in the Redux state; I suppose that's creating just as much as the database info is :laughing:, though we are essentially copying the info from the db.

Reply:
Well change it to create to match the DB convention anyway :smile: